[2023-05-29T15:00:23.978+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: game_alert.postgres_to_s3_task scheduled__2023-05-28T15:00:00+00:00 [queued]>
[2023-05-29T15:00:23.992+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: game_alert.postgres_to_s3_task scheduled__2023-05-28T15:00:00+00:00 [queued]>
[2023-05-29T15:00:23.992+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-05-29T15:00:23.993+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-05-29T15:00:23.993+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-05-29T15:00:24.011+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): postgres_to_s3_task> on 2023-05-28 15:00:00+00:00
[2023-05-29T15:00:24.019+0000] {standard_task_runner.py:55} INFO - Started process 143 to run task
[2023-05-29T15:00:24.027+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'game_alert', 'postgres_to_s3_task', 'scheduled__2023-05-28T15:00:00+00:00', '--job-id', '813', '--raw', '--subdir', 'DAGS_FOLDER/game_alert.py', '--cfg-path', '/tmp/tmpzput_4yg']
[2023-05-29T15:00:24.028+0000] {standard_task_runner.py:83} INFO - Job 813: Subtask postgres_to_s3_task
[2023-05-29T15:00:24.109+0000] {task_command.py:388} INFO - Running <TaskInstance: game_alert.postgres_to_s3_task scheduled__2023-05-28T15:00:00+00:00 [running]> on host b03c83f459ca
[2023-05-29T15:00:24.204+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=game_alert
AIRFLOW_CTX_TASK_ID=postgres_to_s3_task
AIRFLOW_CTX_EXECUTION_DATE=2023-05-28T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-28T15:00:00+00:00
[2023-05-29T15:00:24.223+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-05-29T15:00:24.227+0000] {postgres_to_s3.py:23} INFO - Saved postgres data in text file: games.txt
[2023-05-29T15:00:24.237+0000] {base.py:73} INFO - Using connection ID 'aws_hook' for task execution.
[2023-05-29T15:00:24.238+0000] {connection_wrapper.py:337} INFO - AWS Connection (conn_id='aws_hook', conn_type='aws') credentials retrieved from login and password.
[2023-05-29T15:00:25.882+0000] {postgres_to_s3.py:32} INFO -  /tmp/tmpupdllur5 has been pushed to S3
[2023-05-29T15:00:25.884+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-29T15:00:25.896+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=game_alert, task_id=postgres_to_s3_task, execution_date=20230528T150000, start_date=20230529T150023, end_date=20230529T150025
[2023-05-29T15:00:25.921+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-05-29T15:00:25.958+0000] {taskinstance.py:2585} INFO - 1 downstream tasks scheduled from follow-on schedule check
